### ISSUE TITLE
Show ping result next to favorite instances

### ISSUE THREAD

**Issue by Parrot7483** [libredirect/libredirect#680]
When clicking "**Ping instances**" in the settings for one of the sites the result are shown next to URL's of the instances. 

It would be helpful if the same results are also be shown next to the URL's under "**Add your favorite instances**". This way one can faster decide which instances to remove. 

It would also make sense to test the instances under "**Add your favorite instances**" first.

**Comment by gjedeer** [comment]
Also - add a button to remove dead instances from favourites, if the ping fails. 

The instances go down pretty regularly (RIP pussthecat) and it's quite a chore to upkeep the instance list.

**Comment by ManeraKai** [comment]
@gjedeer

Correct, but instances aren't huge companies or something. And after so many problems, we decided to just "give information" and not touch the instances list ourselves. We provide `cloudflare` and `ping` information but we don't interfere.

**Comment by ManeraKai** [comment]
@Parrot7483 I added the Latency Test to the favorite list too. It will appear in the next release :)

**Comment by gjedeer** [comment]
@ManeraKai I don't think you understood me. I'm not complaining about the instances list, good job compiling it.

I'm asking for a button to mass-remove all instances for which the ping failed from favorites.

Unless it's me who understood your answer the wrong way :)

**Comment by ManeraKai** [comment]
> Unless it's me who understood your answer the wrong way :)

I think so :'D

I the upcoming days we'll shift focus more to [Frontends Manager](https://github.com/libredirect/frontends_manager) than trying to figure out which instance is the best or the most trustworthy.
